change/@microsoft-teams-js-a5218507-221a-43fb-9564-a80174a8f210.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added optional field `messageId` to open `stageView` in new modes if supported by host",
```

```suggestion
  "comment": "Added optional field `messageId` to `stageView.open`",
```

This should also include a brief description of what setting `messageId` actually does